### ISSUE TITLE
Pytorch implementation of this paper

### ISSUE THREAD

**Issue by jpainam** [layumi/2016_person_re-ID#19]
Hi!
Is there anyone who has implemented this paper in pytorch?
Pls kindly share.
Thank.

**Comment by layumi** [comment]
Hi @jpainam ,
Sorry for the late response.
I may work on it after CVPR submission.

For the time being, you may refer to the triplet loss (https://github.com/layumi/Person-reID-triplet-loss). I left an api to add the softmax loss easily. 
And then you could easily modify the code to add verification loss (following the triplet loss).

**Comment by jpainam** [comment]
Thank you for your reply.

**Comment by swg209** [comment]
@jpainam Did you implemented the siamese network?  How to load data in training ? Pls help, thx.

**Comment by jpainam** [comment]
@swg209 [here](https://github.com/layumi/Person-reID-verification) the pytorch implementation of the paper. You can find the triplet mining strategy for how to load data for a siamese network. Hope it helps
